tests/shared/UserLogonRequestTestDefs.js

```diff
@@ -12,30 +12,151 @@
 
 "use strict";
 
-var fluid = require("infusion"),
+var fluid  = require("infusion"),
```

Comment:
Anomalous spacing appeared here

Reply:
It's aligned in the way I often do, and is consistent with itself, but I see that the rest of the test defs follow another style.  I'll change it.